engines/stark/console.cpp

```diff
@@ -468,17 +470,29 @@ bool Console::Cmd_DumpLocation(int argc, const char **argv) {
 }
 
 bool Console::Cmd_ListInventory(int argc, const char **argv) {
-	if (StarkGlobal->getInventory()) {
-		StarkGlobal->printInventory(argc != 2);
-	} else {
+	Resources::KnowledgeSet *inventory = StarkGlobal->getInventory();
+
+	if (!inventory) {
 		debugPrintf("The inventory has not been loaded\n");
+		return true;
+	}
+
+	bool printAll = argc != 2;
```

Comment:
I kept this parameter as before but I am really confused about its meaning. The whole "adding an argument after `listInventory` will print out enabled inventory items" seems kind of weird. Can we change it to having a `-all` argument or a make a new `listAllInventory` command?

Reply:
Sure, feel free to improve this command. Another approach would be to always list all the inventory items and append `(possessed)` or something to the enabled inventory items.